components/concept/awesomebar/src/main/java/mozilla/components/concept/awesomebar/AwesomeBar.kt

```diff
@@ -145,6 +145,10 @@ interface AwesomeBar {
      * It returns a list of [Suggestion]s to be displayed by the [AwesomeBar].
      */
     interface SuggestionProvider {
+        /**
+         * A header title for grouping the suggestions.
+         **/
+        fun groupTitle(): String? = ""
```

Comment:
Can you add a new line before this? It might be better to move this below `id`. It feels like ID is usually one of those starting parameters you would see.